modules/network-observability-quickfilter.adoc

```diff
@@ -11,19 +11,22 @@ Query Options::
 You can use *Query Options* to optimize the search results, as listed below:
 
 ** *Log Type*: The available options *Conversation* and *Flows* provide the ability to query flows by log type, such as flow log, new conversation, completed conversation, and a heartbeat, which is a periodic record with updates for long conversations. A conversation is an aggregation of flows between the same peers.  
-** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for the *Overview* and *Topology* view. The default selected value is *Destination*.
+** *Duplicated flows*: A flow might be reported from several interfaces, and from both source and destination nodes, making it appear in the data several times. By selecting this query option, you can choose to show duplicates. Duplicates are hidden by default. Use the *Direction* filter in the *Common* section in the to switch between ingress and egress traffic.
 ** *Match filters*: You can determine the relation between different filter parameters selected in the advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
 ** *Limit*: The data limit for internal backend queries. Depending upon the matching and the filter settings, the number of traffic flow data is displayed within the specified limit.
 
 Quick filters::
 The default values in *Quick filters* drop-down menu are defined in the `FlowCollector` configuration. You can modify the options from console. 
 
 Advanced filters::
-You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
+You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The *Common* section of filters the results based on the selection, such as *Protocol*, *Direction*, *Network latency*, *Packet drop TCP state*, *Packet drop latent cause*, *DNS Latency*, *DNS Id*, *DNS response code*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
```

Comment:
`The *Common* section of filters the results based on ...` => remove `of` ?

Reply:
I'm not sure it's necessary to list the "Common" filters now, actually. We don't list all available filters under "Source" and "Destination", the ones under "Common" aren't more important.